### DISCUSSION TITLE
希望能出一个安卓版的app，我也可以协助开发

### DISCUSSION THREAD

**Discussion by xuejiehsuej** [cloudreve/cloudreve#3205] [General]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by hatsuyuki280** [comment]
v4计划明年Q1 release，现在已经12月中旬了

咱个人（作为用户）可能会比较推荐等v4出来了再开始讨论客户端的开发相关事宜
（毕竟据说不光不开源，还大量重写了）
现在投入app开发的成本对效果可能并不会比较高

**Comment by wnpllrzodiac** [comment]
出了没，想找一个替代 kodbox的

**Comment by xuejiehsuej** [comment]
> v4计划明年Q1 release，现在已经12月中旬了
> 
> 咱个人（作为用户）可能会比较推荐等v4出来了再开始讨论客户端的开发相关事宜 （毕竟据说不光不开源，还大量重写了） 现在投入app开发的成本对效果可能并不会比较高

现在有相关计划了吗，我觉得还，需要加一个扫码的登陆方式

**Comment by hatsuyuki280** [comment]
> > v4计划明年Q1发布，现在已经12月中旬了
> > 咱个人（用户）可能会比较推荐等v4出来了再开始讨论客户端的开发相关系数（毕竟据说不光不开源，还大量重写了）现在投入app开发的成本效果对可能并不会比较高
> 
> 现在有相关的计划了吗，我觉得还需要加一个扫码的登陆方式

咱也觉得这是个好建议

关于v4的进度，以咱的立场（一般用户）来说并不是很合适透露详细信息。。
不过一些人尽皆知的信息咱大概还是可以稍微介绍一下的。。

现在v4已经在内测中了，
最新的消息是说近期将会开始对v3>v4的升级脚本进行测试，如果一切顺利的话今年Q2前期就能够遇到v4的第一个版本了（推测）

已知v4的后端已经被决定不开源了，&暂时也还没听到关于是否有提供官方API文档的确切消息。
因此就现在已知的信息来说，你想整的v4 app 也许需要从抓浏览器的HTTP请求开始。。。

不过考虑到现在的A测群里的人们提出的feature request，有不少人都提出了希望有API文档之类的东西的内容，未来也说不定会不会有官方API文档发布就是了。。
所以现阶段或许还是建议再等等比较好？

**Comment by Yuerchu** [comment]
Cloudreve v4非官方的API文档我已经在写了，目前进度应该是20%左右
我也没有过多的精力来维护这个文档，要不晚一些的时候开个仓库大家一起维护吧
我现在就本地搭建，然后在前端操作看请求

![Cloudreve v4 Routers](https://github.com/user-attachments/assets/cbf0f286-25cc-4ad5-b262-583fb5c38c98)

**Comment by Yuerchu** [comment]
> Cloudreve v4非官方的API文档我已经在写了，目前进度应该是20%左右 我也没有过多的精力来维护这个文档，要不晚一些的时候开个仓库大家一起维护吧 我现在就本地搭建，然后在前端操作看请求
> 
> ![Cloudreve v4 Routers](https://github.com/user-attachments/assets/cbf0f286-25cc-4ad5-b262-583fb5c38c98)

我开了一个仓库，用来学习交流 Cloudreve 官方文档中没有提到的内容：[https://github.com/Yuerchu/Cloudreve_Docs](https://github.com/Yuerchu/Cloudreve_Docs)

**Comment by hatsuyuki280** [comment]
也不知道你们的这个讨论串后来有没有新的进展了
